library.properties

```diff
@@ -1,9 +1,9 @@
 name=LibTeleinfo
-version=1.1.4
-author=Charles-Henri Hallard <hallard.me>
-maintainer=Charles-Henri Hallard <community.hallard.me>
+version=1.1.5
+author=François-Régis Colin
+maintainer=François-Régis Colin
 sentence=Decoder for Teleinfo (aka TIC) from French smart power meters
 paragraph=This is a generic Teleinfo (aka TIC) French Meter Measure Library, it can be used on Arduino, Particle, ESP8266, Raspberry PI or anywhere you can do Cpp coding.
 category=Communication
-url=https://github.com/hallard/LibTeleinfo
+url=https://github.com/FrColin/LibTeleinfo.git
 architectures=*
```

Comment:
Would have fair leaving original library author when doing a PR

Reply:
You are extremly rigth a made this change by mistake